apps/shell/src/main.tsx

```diff
@@ -9,6 +9,6 @@ const root = ReactDOM.createRoot(
 
 root.render(
   <StrictMode>
-    <App />
+    <App />,
```

Comment:
Looks like a comma snuck in here. This will give every page a comma at the end of it.

```suggestion
    <App />
```

Reply:
oops😅